jade/js/jade.js

```diff
@@ -162,96 +167,90 @@ function showDashboard() {
+    $("#exit-button").click(function () {
+        $("#exit-menu").parent().find("div").slideToggle("slow");
     });
 
-    $("nav a, #clear-search").hover(function() {
-      $(this).addClass("underline pointer");
-    }, function() {
-      $(this).removeClass("underline pointer");
+    $("nav a, #clear-search").hover(function () {
+        $(this).addClass("underline pointer");
+    }, function () {
+        $(this).removeClass("underline pointer");
     });
 
     var red = "#f44336";
     var pink = "#e91e63";
     var blue = "#2196f3";
     var green = "#4caf50";
     var orange = "#ff9800";
-    
+
+    var background = $("#background");
+
     var colors = [red, blue, pink, green, orange];
 
     // apply colors to application container background
-    $('.application-box').css('background-color', function(index) {
-      return colors[index % colors.length];
+    application_box.css("background-color", function (index) {
+        return colors[index % colors.length];
     });
 
-    $(".dash-btn").click(function() {
-      if ($(this).hasClass("dash-btn")) {
-    	  $(this).fadeOut();
-    	}
-      emptyClass("#background");
-      showDashboard();
-      savedSearches();
+    $(".dash-btn").click(function () {
+        if ($(this).hasClass("dash-btn")) {
+            $(this).fadeOut();
+        }
+        emptyClass("#background");
+        showDashboard();
+        savedSearches();
     });
 
     // detect when arrow animation ends
-    $(".applications-arrow-highlight").on( 'webkitAnimationEnd', function() {
-    $(this).fadeOut();
+    $(".applications-arrow-highlight").on('webkitAnimationEnd', function () {
+        $(this).fadeOut();
     });
 
-      // backgrounds
-    $("li.application-category").mouseover(function() {
-      emptyClass("#background");
-    	$(".dash-btn").fadeIn();
-
-      if ($(this).hasClass("office")) {
-        $("#background").addClass("office-background");
-
-      } else if ($(this).hasClass("development")) {
-        $("#background").addClass("development-background");
-
-      } else if ($(this).hasClass("education")) {
-        $("#background").addClass("education-background");
-        
-      } else if ($(this).hasClass("multimedia")) {
-        $("#background").addClass("multimedia-background");
-
-      } else if ($(this).hasClass("gaming")) {
-        $("#background").addClass("games-background");
-
-      } else if ($(this).hasClass("graphics")) {
-        $("#background").addClass("graphics-background");
-
-      } else if ($(this).hasClass("go-online")) {
-        $("#background").addClass("internet-background");
-        
-      } else if ($(this).hasClass("accessories")) {
-        $("#background").addClass("accessories-background");
-      } 
-      savedSearches();
+    // backgrounds
+    $("li.application-category").mouseover(function () {
+        emptyClass("#background");
+        $(".dash-btn").fadeIn();
+        var hasClass = $(this).hasClass;
```

Comment:
this is not working, apparently only happens in webkit browsers
  
[Error] TypeError: undefined is not an object (evaluating 'this[d++]')
	hasClass (jquery-3.1.1.min.js:4:3306)
	(anonymous function) (jade.js:225)
	dispatch (jquery-3.1.1.min.js:3:10321)